package/.pylintrc

```diff
@@ -199,7 +199,6 @@ enable=abstract-class-instantiated,
        print-statement,
        xrange-builtin,
        zip-builtin-not-iterating,
-       map-builtin-not-iterating,
```

Comment:
Look bellow. It probably fails because you use `+` on `map`. In python 3, `map` is an iterator so you cannot use `+` on it. You need to convert the map output to a list to extend it.

By the way, if you use `map`, you should import it from `six`.

In general, you should never remove checks from pylint. If you understand exactly why the test fail, and you are sure about why you are doing, you may deactivate the check locally.